### ISSUE TITLE
CC-CDS - Complete Dash Set 1.0.10.3 doesn't work on mobile

### ISSUE THREAD

**Issue by djmartinvega** [SHWotever/SimHub#417]
Hi! Have you any idea why last version of EnsiFerrum CC-CDS dash set for ACC from Racedepartment stuck on loading screen on mobile? In my case it's iOS but I saw that other users have similar problems on Android devices. 
Any other dashes work well on same mobile. Also I have checked on two different browser - safari and Chrome for iOS.

https://www.racedepartment.com/downloads/acc-cds-complete-dash-set.24736/

**Comment by SHWotever** [comment]
Hi ! 

Ah the ensy dash ! I'm admirative of the patience and details he put in it. But I'm afraid this dash is also the heaviest I've ever seen, it is loading about 340 widgets while typical simhub dashs usually contains 1 to 10 of them at best

Due to that, It is reaching the limits of the web engine model I currently use which is "preloading everything" to ensure the fastest screen switching possible.  About this point, Safari is known to have very bad memory management and will indeed fail easily on a so big dashboard. About android, chrome manages it better but it may still fail if the phone does not have enough memory. I tried it on an Honor 9 and it loaded perfectly (except that it took more time than usual). 

Obviously this dash is a specific case pushing far beyond the capacities I've designed the web engine for. Only a full rewrite taking this particular case into account, for instance by only loading the current screen and unloading everything else would allow to work around that. But it would certainly degrade the experience for all other more modest dashes by making screen switch way slower, which feels like a very bad compromise in my opinion.
